delfin/drivers/dell_emc/scaleio/rest_handler.py

```diff
@@ -0,0 +1,397 @@
+        except Exception as e:
+            LOG.error("Get Storage disk error: %s", six.text_type(e))
+            raise exception.InvalidResults(e)
+
+    def list_alerts(self, query_para=None):
+        alert_list = []
+        try:
+            storage_alert = self.get_rest_info(consts.REST_SCALEIO_ALERT)
+            for json_alert in (storage_alert or []):
+                match_key = json_alert.get('id') + json_alert.get('name')
+                occur_time = json_alert.get('startTime')
+                datetime_obj = datetime.datetime.strptime(
+                    occur_time, consts.DATETIME_UTC_FORMAT)
+                alert_time = int(time.mktime(datetime_obj.timetuple()) *
+                                 consts.DEFAULT_ALERTS_TIME_CONVERSION
+                                 + datetime_obj.microsecond /
+                                 consts.DEFAULT_ALERTS_TIME_CONVERSION)
+                if not alert_util.is_alert_in_time_range(query_para,
+                                                         alert_time):
+                    continue
+                alert_type = json_alert.get('alertType')
+                alert_model = {
+                    'alert_id': json_alert.get('id'),
+                    'alert_name': alert_type + json_alert.get('name'),
+                    'severity': json_alert.get('severity'),
+                    'category': constants.Category.FAULT,
+                    'type': alert_type,
+                    'sequence_number': json_alert.get('uuid'),
+                    'description': json_alert.get('alertType'),
+                    'occur_time': alert_time,
+                    'match_key': hashlib.md5(
+                        match_key.encode()).hexdigest()
+                }
+                alert_list.append(alert_model)
+            return alert_list
+        except exception.DelfinException as err:
+            err_msg = "Get Storage alerts error: %s" % err.msg
+            LOG.error(err_msg)
+            raise err
+        except Exception as e:
+            LOG.error("Get Storage alerts error: %s", six.text_type(e))
+            raise exception.InvalidResults(e)
+
+    def list_storage_host_initiators(self, storage_id):
+        initiators_list = []
+        try:
+            storage_initiators = self.get_rest_info(
+                consts.REST_SCALIO_INITIIATORS)
+            for initiators_json in (storage_initiators or []):
+                status = initiators_json.get('mdmConnectionState')
+                initiators_id = initiators_json.get('id')
+                initiators_type = constants.InitiatorType.UNKNOWN
+                if 'iscsi' in initiators_json.get('perfProfile'):
+                    initiators_type = constants.InitiatorType.ISCSI
+                if 'Connected' == status:
+                    status = constants.HostStatus.NORMAL
+                elif 'Disconnected' == status:
+                    status = constants.HostStatus.OFFLINE
+                initiators_dict = {
```

Comment:
pls add the  initiator type.
![image](https://user-images.githubusercontent.com/59107077/172134847-6d1c6835-7da4-408b-a76e-dbd5e019d955.png)

Reply:
Added the initiator type and modified the initiator status
![image](https://user-images.githubusercontent.com/88076879/172275997-868d9d8a-c738-45ae-8194-cbfa8239b27d.png)